integrations/terraform-cloud/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,69 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: workspace
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .attributes.name
+          blueprint: '"terraformWorkspace"'

```

Comment:
you did not update the blueprints identifiers
please make sure the integrations works end to end